static/openapi/prerendering-spec.json

```diff
@@ -3060,1025 +3384,1091 @@
-                    "type": "string",
-                    "description": "Text in the rendered page HTML that identifies a not found page (404). If the rendered page includes this text, then the HTTP status code of the response will be 404. For example, if your not found page contains the text, \"Sorry, the page you were looking for could not be found.\", then this text can be used."
-                  },
-                  "allowOriginHeaderValue": {
+                  "cronSchedule": {
                     "type": "string",
-                    "description": "Value of x-photoniq-prerender-request header."
-                  },
-                  "passThroughHeaders": {
-                    "type": "array",
-                    "items": { "type": "string", "description": "Header key." },
-                    "description": "List of header keys that are forwarded from the bot request to the origin website."
-                  },
-                  "addResponseHeaders": {
-                    "type": "object",
-                    "additionalProperties": {
-                      "type": "object",
-                      "properties": {
-                        "basedOnStatusCode": {
-                          "type": "boolean",
-                          "description": "Whether the header value should be based on the response status code."
-                        },
-                        "value": {
-                          "oneOf": [
-                            {
-                              "type": "string",
-                              "description": "Header value."
-                            },
-                            {
-                              "type": "object",
-                              "description": "Status code. Ranges like 2xx can be also used.",
-                              "additionalProperties": {
-                                "type": "string",
-                                "description": "Header value."
-                              }
-                            }
-                          ]
-                        }
-                      },
-                      "required": ["basedOnStatusCode", "value"],
-                      "description": "Additional response headers configuration."
-                    },
-                    "description": "Map of headers to be added to the response."
-                  },
-                  "caching": {
-                    "type": "boolean",
-                    "description": "Whether or not to cache rendered pages."
+                    "description": "Cron expression for scheduling."
                   },
-                  "ttl": {
-                    "type": "number",
-                    "description": "Cache lifetime of rendered pages (in seconds)."
+                  "isRenderForMobile": {
+                    "type": "boolean",
+                    "description": "Whether to render for mobile devices or not."
```

Comment:
```suggestion
                    "description": "Determining if rendering should be tailored for mobile devices."
```

Reply:
Not sure about this one.  But these "or not" are always strange.

Reply:
If you make any changes here, make them for all the places that use this wording.

Reply:
I improved all messages with `or not`